test/datafmt.jl

```diff
@@ -204,7 +204,7 @@ let i18n_data = ["Origin (English)", "Name (English)", "Origin (Native)", "Name
         "Russia", "Mikhail Gorbachev", "Россия", "Михаил Горбачёв",
         "Russia", "Boris Grebenshchikov", "Россия", "Борис Гребенщиков",
         "Slovenia", "\"Frane \"\"Jezek\"\" Milcinski", "Slovenija", "Frane Milčinski - Ježek",
-        "Syracuse (Sicily)", "Archimedes", "Συρακούσα", "Ἀρχιμήδης",
+        "Syracuse (Sicily)", "Archimedes", "Συρακούσα", "Ἀρχιμήδης",
```

Comment:
Probably Unicode normalization change – both strings have accents which have composed and decomposed forms. Doesn't really matter which we use, so this is a fairly harmless change.